en/docs/guides/password-mgt/challenge-question.md

```diff
@@ -1,17 +1,183 @@
+WSO2 Identity Server enables resetting user passwords by correctly responding to predefined challenge questions (also known as security questions).
 
-{!TODO: insert-fragment!} 
+## Prerequisites
+
+**Create a user**
+
+1. Log in to the WSO2 Identity Server Management Console (`https://<IS_HOST>:<PORT>/carbon`) using administrator credentials (`admin:admin`).
+    
+2. Navigate to **Main** > **Identity** > **Users and Roles** > **Add**.
+    
+3. Click **Add New User** and create a new user with the username "Alex".
+    
+4. Click **Finish**.
+
+## Enable password reset via Challenge Questions
+
+Follow the steps below to configure WSO2 Identity Server to enable password reset by answering to a challenge question.  
+
+1.  Sign in to the WSO2 Identity Server Management Console (`https://<HOST>:<PORT>/carbon`) as an administrator. 	  
+
+2.	On the **Main** menu of the Management Console, click **Identity > Identity Providers > Resident**.
+
+	![resident-idp](../../../assets/img/fragments/resident-idp.png)  
+
+3.	Under the **Account Management** section, click **Account Recovery**.
+
+	![account-recovery-option](../../../assets/img/fragments/account-recovery-option.png)  	
+
+4.	Enter the required values as given below:
+
+	-	**Security question based password recovery**: Selected
+
+	-	**Number of questions required for password recovery**: `2` 
+	
+	![security-question-based-password-recovery-option](../../../assets/img/guides/security-question-based-password-recovery-option.png)
+
+5.	Click **Update**. 
+
+----
+
+## Configure the challenge questions
+
+1. Access WSO2 Identity Server My Account (`https://<HOST>:<PORT>/myaccount`) application.		
+
+2.	Log in with the credentials of the user account that you created.
+
+	![myaccount-login](../../../assets/img/fragments/myaccount-login.png)
+ 
+3. Select `Security` tab.
+
+    ![myaccount-security-tab](../../../assets/img/fragments/myaccount-security-tab.png)
+
+3.	Under **Account Recovery**, click **+** to add or update security questions.
+
+	![myaccount-recovery-section](../../../assets/img/fragments/myaccount-recovery-section.png)
+
+4.	Configure the challenge questions as given below:
+
+	-	**Challenge Question 1**: `Name of your first pet?`
+	-	**Your Answer**: `Tommy`
+	-	**Challenge Question 2**: `Favourite food?`
+	-	**Your Answer**: `Pizza`
+
+    ![myaccount-security-questions-form](../../../assets/img/guides/my-account/account-recovery/security-questions-form.png)
+    
+5. Click **Save**.
+
+6. Sign out.  
 
 ---
 
-## Manage challenge questions using REST APIs
+## Recover password using the My Account application
+
+1. Access WSO2 Identity Server My Account (`https://<HOST>:<PORT>/myaccount`) application.
+
+2.	Click **Password**.
+
+    ![forgotten-password-option](../../../assets/img/guides/forgotten-password-option.png)
+
+3.	Enter the user name as `Alex` and if multiple recovery options are displayed, select the **Recover with Security Questions** option.
```

Comment:
IMO, we should not be giving users the definite values to be entered. It becomes more like a quick start that way. 

Shall we give more generic instructions and then maybe have a try it out section where we give this as an example? WDYT?

Reply:
removed 'Alex'